docs/README.md

```diff
@@ -29,40 +27,40 @@ For support and questions, visit [the NeoForged Discord server][Discord].
 
 # Creating Mods
 
-[See the "Getting Started" section in the NeoForged Documentation][Getting-Started].
+[See the "Getting Started" section in the NeoForge Documentation][Getting-Started].
 
-# Contribute to Forge
+# Contribute to NeoForge
 
-If you wish to actually inspect Forge, submit PRs or otherwise work
- with Forge itself, you're in the right place!
+If you wish to actually inspect NeoForge, submit PRs or otherwise work
+ with NeoForged itself, you're in the right place!
 
- [See the guide to setting up a Forge workspace][ForgeDev].
+ [See the guide to setting up a NeoForge workspace][NeoForgeDev].
 
 ### Pull requests
 
-[See the "Making Changes and Pull Requests" section in the Forge documentation][Pull-Requests].
+[See the "Making Changes and Pull Requests" section in the NeoForge documentation][Pull-Requests].
 
 Please read the contributing guidelines found [here][Contributing] before making a pull request.
 
 ### Contributor License Agreement
-We require all contributors to acknowledge the [Neoforged Contributor License Agreement][CLA]. 
+We require all contributors to acknowledge the [NeoForge Contributor License Agreement][CLA].
 Please ensure you have a valid email address associated with your GitHub account to do this. If you have previously 
  signed it, you should be OK.
 
 #### Donate
-*NeoForged is a large project with many collaborators working on it around the clock. It will always remain 
+*NeoForge is a large project with many collaborators working on it around the clock. It will always remain 
```

Comment:
This section should probably be rewritten, as we currently don't have a way of accepting donations - that is still a work in progress.

Reply:
Reworded in a hopefully good way